src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -505,6 +634,168 @@ async def chat_completion(
+                        raise BadRequest(
+                            "Model configuration is required when previewing an existing "
+                            "LLM evaluator. Please provide the 'model' field."
+                        )
+
+                    async with info.context.db() as session:
+                        llm_evaluator_orm = await session.get(models.LLMEvaluator, db_id)
+                        if llm_evaluator_orm is None:
+                            raise BadRequest(f"LLM evaluator with id {evaluator_id} not found")
+
+                        prompt_version_tag_id = llm_evaluator_orm.prompt_version_tag_id
+                        if prompt_version_tag_id is not None:
+                            prompt_version_query = (
+                                select(models.PromptVersion)
+                                .join(models.PromptVersionTag)
+                                .where(models.PromptVersionTag.id == prompt_version_tag_id)
+                            )
+                        else:
+                            prompt_version_query = (
+                                select(models.PromptVersion)
+                                .where(
+                                    models.PromptVersion.prompt_id == llm_evaluator_orm.prompt_id
+                                )
+                                .order_by(models.PromptVersion.id.desc())
+                                .limit(1)
+                            )
+                        prompt_version = await session.scalar(prompt_version_query)
+                        if prompt_version is None:
+                            raise BadRequest(
+                                f"Prompt version not found for evaluator {evaluator_id}"
+                            )
+
+                        llm_client = await get_playground_client(
+                            preview_item.model,
+                            info.context.db,
+                            info.context.decrypt,
+                        )
+
+                        evaluator = LLMEvaluatorRunner.from_orm(
+                            llm_evaluator_orm=llm_evaluator_orm,
+                            prompt_version_orm=prompt_version,
+                            llm_client=llm_client,
+                        )
+
+                        for context in prepared_contexts:
+                            eval_result = await evaluator.evaluate(
+                                context=context,
+                                input_mapping=input_mapping,
+                            )
+                            context_results.append(_to_annotation(eval_result))
+
+                elif type_name == CodeEvaluator.__name__:
+                    raise BadRequest("Code evaluators are not yet supported for preview")
+                else:
+                    raise BadRequest(f"Unknown evaluator type: {type_name}")
+
+            elif evaluator_input.inline_llm_evaluator is not strawberry.UNSET:
```

Comment:
```suggestion
            elif evaluator_input.inline_llm_evaluator:
```

We are no checking for `None` as we did in the previous conditional above.